pkg/server/version.go

```diff
@@ -17,14 +17,35 @@ limitations under the License.
 package server
 
 import (
-	"errors"
+	"fmt"
 
+	"github.com/golang/protobuf/ptypes/empty"
 	"golang.org/x/net/context"
 
 	runtime "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1"
 )
 
+const (
+	containerName        = "containerd"
+	containerdAPIVersion = "0.0.0"
+	containerdVersion    = "0.0.0"
+	// kubeAPIVersion is the api version of kubernetes.
+	kubeAPIVersion = "0.1.0"
```

Comment:
Couple questions.. 1) Should / could this be set at build vs being hard coded? 2) Why kubeAPIVersion and not kubelet and / or kubelet/cri?